main.js

```diff
@@ -37,6 +37,12 @@ if (typeof(PLAYER)!="object" || typeof(PLAYER.nickname) != "string") {
     throw("Page doesn't have player data, but you are logged in.");
   }
   // FIXME: handle nia takedown in progress
+  
+  // add login form stylesheet
```

Comment:
Should we change style for desktop IITC?

Reply:
Well, why not? CSS `media` can create an adaptive view for all screen resolutions

Reply:
Ok. I have no objections (though not seen new design yet).